test/merchant_test.rb

```diff
@@ -35,10 +35,25 @@ def test_find_all_items_by_merchant_id
   def test_find_all_invoices_by_merchant_id
     merchant = Merchant.new({:id => "99", :name => "John"}, parent)
     result = merchant.find_invoices_by_merchant_id(99)
-    assert_equal 2, result.size 
+    assert_equal 2, result.size
     refute_equal 4, result.size
     merchant2 = Merchant.new({:id => "2", :name => "John"}, parent)
     result2 = merchant2.find_invoices_by_merchant_id(2)
-    assert_equal 2, result2.size 
+    assert_equal 2, result2.size
+  end
+
+  #def test_find_transaction_by_way_of_merchant_id
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.